modules/users/db/schema.prisma

```diff
@@ -4,8 +4,24 @@ datasource db {
 }
 
 model User {
-    id        String   @id @default(uuid()) @db.Uuid
-    username  String   @unique
-    createdAt DateTime @default(now())
-    updatedAt DateTime @updatedAt
+    id			String		@id @default(uuid()) @db.Uuid
+    username	String		@unique
+    createdAt	DateTime	@default(now())
+    updatedAt	DateTime	@updatedAt
+
+	pfpId		String? 	@db.Uuid
+	pfp			Pfp?
+}
+
+model Pfp {
+	uploadId	String		@id @db.Uuid
+	url			String
+	urlExpiry	DateTime
+
+	userId		String		@db.Uuid @unique
+	user		User		@relation(fields: [userId], references: [id])
+
+	createdAt	DateTime	@default(now())
+	updatedAt	DateTime	@updatedAt
+	finishedAt	DateTime?
 }
```

Comment:
This data is all stored in uploads, is there a reason to duplicate this?

Reply:
Presigned URLs don't touch API endpoints, they're done locally cryptographically

Reply:
To clarify: this should be a single column in `User` for `avatarUploadId`